src/graphql/call/fragment.call.graphql

```diff
@@ -23,4 +24,28 @@ fragment call on Call {
       ...basicUserDetails
     }
   }
+  proposalWorkflow {
```

Comment:
@martin-trajanovski Change #1 I added proposalWorkflow on the call fragment. I was wondering is this too expensive to fetch? If it is, then I'll make a task to refactor a little.

Reply:
@jekabs-karklins I see that on the call we don't need `proposalWorkflowConnectionGroups`. What we need is just id, name and description I guess. I think that is just what I added.